profile/README.md

```diff
@@ -0,0 +1,19 @@
+DBOS provides lightweight, durable, Postgres-backed workflows.
```

Comment:
Could we put a little bit of "why DBOS" diagrams in there?

Reply:
Too big. Want them to see (and star!) our repos.